src/kit.ts

```diff
@@ -22,6 +22,14 @@ const localize: nls.LocalizeFunc = nls.loadMessageBundle();
 
 const log = logging.createLogger('kit');
 
+/**
```

Comment:
@bobbrow I think it makes sense to replace these special kit variables with a enum. Also do we really need to have the type `export type scanforkitsKitType = '__scanforkits__';`

Reply:
I don't have a strong opinion here.

Reply:
Well I did convert these to an enum in my latest commit.